test/Datadog.Trace.Tests/SpanTests.cs

```diff
@@ -135,5 +138,28 @@ public void Accurate_Duration(int minimumSleepMilliseconds)
                 stopwatchSpanDiff < maxDifference,
                 $"Span duration difference ({stopwatchSpanDiff}ms) outside of allowed threshold {maxDifference}ms. Expected average: {avgStopwatch}ms, actual average: {avgSpan}ms");
         }
+
+        [Fact]
+        public void TopLevelSpans()
+        {
+            var spans = new List<(Scope Scope, bool IsTopLevel)>();
+
+            spans.Add((_tracer.StartActive("Root", serviceName: "root"), true));
+            spans.Add((_tracer.StartActive("Child1", serviceName: "root"), false));
+            spans.Add((_tracer.StartActive("Child2", serviceName: "child"), true));
+            spans.Add((_tracer.StartActive("Child3", serviceName: "child"), false));
+            spans.Add((_tracer.StartActive("Child4", serviceName: "root"), true));
+
+            // Dispose the scope to populate the top-level property
+            foreach (var (scope, _) in spans.AsEnumerable().Reverse())
```

Comment:
You don't need to dispose these anymore to set the top-level property, do you?

Reply:
I was going to ask the same question, but then thought it doesn't hurt (although the comment may be wrong). Thanks for asking 😅

Reply:
You're right. Removed in https://github.com/DataDog/dd-trace-dotnet/pull/1303/commits/87ab80ddc9107b9b6cff3028a319b5824358d6e7